rerun_py/rerun_bindings/rerun_bindings.pyi

```diff
@@ -927,3 +930,341 @@ def asset_video_read_frame_timestamps_nanos(
+
+            The original view will not be modified.
+
+        """
+
+    def filter_range_nanos(self, start: int, end: int) -> Self:
+        """
+        Filter the view to only include data between the given index values expressed as seconds.
+
+        This range is inclusive and will contain both the value at the start and the value at the end.
+
+        The view must be of a temporal index type to use this method.
+
+        Parameters
+        ----------
+        start : int
+            The inclusive start of the range.
+        end : int
+            The inclusive end of the range.
+
+        Returns
+        -------
+        RecordingView
+            A new view containing only the data within the specified range.
+
+            The original view will not be modified.
+
+        """
+
+    def filter_index_values(self, values: IndexValuesLike) -> Self:
+        """
+        Filter the view to only include data at the provided index values.
+
+        The index values returned will be the intersection between the provided values and the
+        original index values.
+
+        This requires index values to be a precise match. Index values in Rerun are
+        represented as i64 sequence counts or nanoseconds. This API does not expose an interface
+        in floating point seconds, as the numerical conversion would risk false mismatches.
+
+        Parameters
+        ----------
+        values : IndexValuesLike
+            The index values to filter by.
+
+        Returns
+        -------
+        RecordingView
+            A new view containing only the data at the specified index values.
+
+            The original view will not be modified.
+
+        """
+
+    def filter_is_not_null(self, column: AnyComponentColumn) -> Self:
+        """
+        Filter the view to only include rows where the given component column is not null.
+
+        This corresponds to rows for index values where this component was provided to Rerun explicitly
+        via `.log()` or `.send_columns()`.
+
+        Parameters
+        ----------
+        column : AnyComponentColumn
+            The component column to filter by.
+
+        Returns
+        -------
+        RecordingView
+            A new view containing only the data where the specified component column is not null.
+
+            The original view will not be modified.
+
+        """
+
+    def using_index_values(self, values: IndexValuesLike) -> Self:
+        """
+        Replace the index in the view with the provided values.
+
+        The output view will always have the same number of rows as the provided values, even if
+        those rows are empty. Use with [`.fill_latest_at()`][rerun.dataframe.RecordingView.fill_latest_at]
+        to populate these rows with the most recent data.
+
+        This requires index values to be a precise match. Index values in Rerun are
+        represented as i64 sequence counts or nanoseconds. This API does not expose an interface
```

Comment:
as above, I'm confused about the impliations of the recent time refactors here

Reply:
Basically none? Again, this makes this API relatively low-level and, frankly, improvable. For starters, it should be compatible with the index column with use with `rr.send_column()`.